site-src/implementation-table.md

```diff
@@ -0,0 +1,25 @@
+The following tables are populated from the conformance reports uploaded by project implementations. They are separated into the extended features that each project supports listed in their reports.
```

Comment:
How about something like this? Please feel free to wordsmith here, but I think this covers the main things: only Core-conformant implementations are listed here, and this is still being worked on (although I should say again that I think that @xtineskim has done a great job here.)

```suggestion
The following tables are populated from the conformance reports uploaded by project implementations. They are separated into the extended features that each project supports listed in their reports.

Implementations only appear in this page if they pass Core conformance for the resource type, and the features listed should be Extended features.

!!! warning+ "Comparison under active development"
    This page is under active development and is not in its final form, especially for the project name and the names of the features.
    However, as it is based on submitted conformance reports, the information is correct.
```

Reply:
added this into the portion of the python file that spits out this markdown! Thank you for the suggestion @youngnick  ! 🖊️ 🙏